cmake/emu/qemu.cmake

```
@@ -257,8 +257,8 @@ if(CONFIG_X86_64)
     COMMAND
     ${CMAKE_OBJCOPY}
     -j .locore
-    ${CMAKE_BINARY_DIR}/zephyr-qemu.elf
-    ${CMAKE_BINARY_DIR}/zephyr-qemu-locore.elf
+    ${CMAKE_BINARY_DIR}/zephyr/zephyr-qemu.elf
+    ${CMAKE_BINARY_DIR}/zephyr/zephyr-qemu-locore.elf
```

We have `ZEPHYR_BINARY_DIR` for this purpose.
```suggestion
    ${ZEPHYR_BINARY_DIR}/zephyr-qemu-locore.elf
```